zerver/webhooks/gitlab/view.py

```diff
@@ -213,6 +213,34 @@ def get_commented_commit_event_body(payload: WildValue, include_title: bool) ->
     )
 
 
+def get_commented_design_event_body(payload: WildValue, include_title: bool) -> str:
+    comment = payload["object_attributes"]
+    repository_url = payload["repository"]["homepage"]
+    new_path = comment["original_position"]["new_path"].tame(check_string)
+    assert isinstance(new_path, str)
+    new_path_split = new_path.split("/")
```

Comment:
Since we using some manual transformations to fetch info from the payload. I think a comment hear mentioning the structure of `new_path` would really help others to easily grasp what's happening here.